contracts/Comptroller.sol

```diff
@@ -313,10 +313,12 @@ contract Comptroller is ComptrollerV5Storage, ComptrollerInterface, ComptrollerE
         // Shh - currently unused
         cToken;
         redeemer;
+        redeemAmount;
+        redeemTokens;
 
-        // Require tokens is zero or amount is also zero
```

Comment:
The immutable tokens still need this, don't think we can get rid of redeemVerify until all cTokens are upgraded to not need it